modules/account-lib/test/unit/coin/celo/transactionBuilder/staking.ts

```diff
@@ -2,6 +2,7 @@ import should from 'should';
 import { coins } from '@bitgo/statics';
 import { Celo, getBuilder } from '../../../../../src';
 import { StakingOperationTypes, TransactionType } from '../../../../../src/coin/baseCoin';
+import { RLP } from 'ethers/utils';
```

Comment:
Why the extra dependency if we don't reference it?

Reply:
oh, oops - I was using it for debugging. Will remove in a separate cleanup PR